docs/api/api/api.txt

```diff
@@ -1953,6 +1953,11 @@ XmlResult: status_ok
 
 DELETE /staging/<staging_workflow_project>/workflow
   Delete a staging workflow associated to a given project.
+
+Parameters: 
+  with_staging_projects
+
+XmlBody: none
```

Comment:
I don't think we should do this. Otherwise, as I said in my previous comments, we need to be consistent and do it for all `DELETE` endpoints.

Reply:
ok. i will add 'XmlBody: none' to the all other 'DELETE' endpoints.